vllm/config.py

```diff
@@ -973,14 +1001,14 @@ class SchedulerConfig:
     """
 
     def __init__(self,
+                 task: Task,
```

Comment:
I try to keep the order of arguments consistent with `EngineArgs`. Since the subsequent arguments are required arguments, I cannot provided a default value to `task` without changing its position.